docs/CarSim.md

```diff
@@ -0,0 +1,74 @@
+# CarSim
+
+The monoDrive Simulator is run with physx car physics, but can be run with CarSim for more optimal car physics. 
```

Comment:
By default, the monoDrive Simulator uses PhysX for vehicle dynamics. Alternatively, monoDrive also supports co-simulation with CarSim.